src/js/controllers/logmood.js

```diff
@@ -0,0 +1,53 @@
+app.controller('LogMoodCtrl', function($scope,sqlService,$ionicPlatform){
+  $ionicPlatform.ready(function() {
+    // initialize database
+    sqlService.init().then((res) => {
+      // run a view query
+      sqlService.viewTable('preferences_table').then(
+        (result) => console.log("View result", result, result.rows.item(0)),
+        (err) => console.log("View error", err)
+      );
+    })
+  });
+
+  $scope.logmood = {
+  $scope.moods = [
+    {id:'1', name:'Anger'},
+    {id:'2', name:'Disgust'},
+    {id:'3', name:'Fear'},
+    {id:'4', name:'Happiness'},
+    {id:'5', name:'Sadness'},
+    {id:'6', name:'Surprise'}
+  ];
+
+  $scope.moodIntensity = [
+      {id:'intensity1', name:'1 - Bad'};
+      {id:'intensity2', name:'2 - Not OK'};
+      {id:'intensity3', name:'3 - Neutral'};
+      {id:'intensity4', name:'4 - Good'};
+      {id:'intensity5', name:'5 - Excellent'};
+  ];
+};
```

Comment:
Lines 13-30 will result in an error.  Can't define scope variables within scope variables.

aka $scope.moods and $scope.moodIntensity can't be defined and thrown inside of $scope.logmood all at once.